ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config

```diff
@@ -41,12 +47,6 @@ set ACL for acs-commons-remote-assets-service
     # If /etc/tags is used, these ACLs be set manually - if RepoInit defines this structure, it supersedes the use of /content/cq:tags
     # allow jcr:read, jcr:versionManagement, rep:write, crx:replicate on /etc/tags
 end
-
-create service user acs-commons-manage-redirects-service with path system/acs-aem-commons
```

Comment:
Was this removed on purpose?

Reply:
it was in here twice, it should be in the new file at line 18

```
# user to invalidate cached redirects on change
create service user acs-commons-manage-redirects-service with path system/acs-aem-commons
set ACL for acs-commons-manage-redirects-service
    allow jcr:read on /
    allow jcr:read on /conf
end
```

Reply:
ah yes now I see you consolidated the two, thanks!